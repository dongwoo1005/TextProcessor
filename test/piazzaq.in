stdin doublewords
hi how are you
sample.txt doublewords
