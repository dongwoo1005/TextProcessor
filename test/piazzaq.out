hi
hi
how
how
are
are
you
you
sample.txt
sample.txt
doublewords
doublewords
